README.adoc

```diff
@@ -128,6 +128,12 @@ The `vertx-junit5` modules allow testing Vert.x asynchronous operations with JUn
 
 See link:junit5-examples/README.adoc[Vert.x JUnit 5 examples].
 
+=== Vert.x Opentracing examples
+
+link:opentracing-examples/README.adoc[Vert.x Opentracing examples] contains a wide range of examples using Vert.x Opentracing.
+
+See link:junit5-examples/README.adoc[Vert.x JUnit 5 examples].
```

Comment:
Mistake? Why a junit5 link in the OT section?

Reply:
yes mistake